README.md

```diff
@@ -1,3 +1,19 @@
 # Northcoders News API
 
-Create two .env files, both for the development and test databases. Inside each file set PGDATABASE = name of the database. Finally, add these two files to .gitignore so that when you push your work on github the databases names are not exposed.
\ No newline at end of file
+# Introduction
+Created a backend news API with different endpoints which is available to view on https://news-project-1.onrender.com.
+
+# Summary 
+The project is a backend news API that has endpoints that we can access to view the data for a given endpoint. This API will feed the front end, providing it with the information contained in the API. 
+
+# Set up instructions
+1. Clone the following githup repository URL https://github.com/Gurtajs/News-project by running git clone https://github.com/<github username>/News-project.git
```

Comment:
GitHub not githup 👀 

You can make text look `all codey` in markdown by surrounding it with backticks, might be useful for formatting your commands